### DISCUSSION TITLE
Can we propose different feed topics?

### DISCUSSION THREAD

**Discussion by ASRagab** [dailydotdev/daily#1049] [Feature Requests]
- Algorithmic Trading
- Computational Finance
- Quantitative Finance
- Trading Strategies
- Trading Bots
- High-Frequency Trading (HFT)

**Comment by idoshamun** [comment]
Hi @ASRagab,

Please check our guide on how to suggest a new source: https://docs.daily.dev/docs/for-content-creators/suggest-new-source
